ginkgo-kubernetes-all.Jenkinsfile

```diff
@@ -176,7 +176,7 @@ pipeline {
                         KUBECONFIG="${TESTDIR}/vagrant-kubeconfig"
```

Comment:
Can a description be added to this commit to provide more context? Some of the changes appear unrelated at a first glance; a succinct description would help clean up confusion.